File: Week_0.1/q14.c
#include <stdio.h>

int main() {
    int i, j, n;
    int a1[50][50], a2[50][50];
    
    printf("Enter size of arrays : ");
    scanf("%d", &n);

    printf("Enter the values of array 1 :");
    for(i = 0; i < n; i++) {
        for(j = 0; j < n; j++) {
            scanf("%d",&a1[i][j]);
        }
    }

    printf("Enter the values of array 2 : ");
    for(i = 0; i < n; i++) {
        for(j = 0; j < n; j++) {
            scanf("%d",&a2[i][j]);
        }
    }

    printf("Sum of 2 arrays is: \n");
    for(i = 0; i < n; i++) {
        for(j = 0; j < n; j++) {
            printf("%d ", a1[i][j] + a2[i][j]);
        }
        printf("\n");
    }
    
}

File: Week_3/q14.js
var readlineSync = require('readline-sync')

let limit = readlineSync.question("Enter the limit : ");

let a = []
let b = []
let sum = []

console.log("Enter numbers");
for(let i = 0; i < limit; i++) {
   a[i] = []
   for(let j = 0; j < limit; j++) {
    a[i][j] = readlineSync.question()
   }
}

console.log("Enter numbers");
for(let i = 0; i < limit; i++) {
   b[i] = []
   for(let j = 0; j < limit; j++) {
    b[i][j] = readlineSync.question()
   }
}

for(let i = 0; i < limit; i++) {
    sum[i] = []
    for(let j = 0; j < limit; j++) {
        sum[i][j] = (Number(a[i][j]) + Number(a[i][j]))

    }
}
console.log(sum);